### PR TITLE
Added agency endpoint

### PR DESCRIPTION
**Pull request by marcDeSantis**

### CODE REVIEW

File: routes/agencies.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  db.agency.find(req.query, function(err, items){
+    return res.json(items);
+  });
+});
+
+router.get('/:id', (req, res, next) => {
+  const id = Number(req.params.id);
+  var db = req.app.get('db');
+  db.agency.find(id, function(err, item){
+    var result = item || {};
```
Comment (stlcubsfan) [inline routes/agencies.js]: Do we want to use 404's in situations like this or empty objects?  Not opposed to either, more important to the front end than anything else

File: routes/agencies.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  var db = req.app.get('db');
+  db.agency.insert(req.body, function(err, item){
+    return res.status(HttpStatus.CREATED).json(item);
+  });
+});
+
+router.put('/:id', (req, res, next) => {
+  var db = req.app.get('db');
+  var agency = req.body;
+  item.id = Number(req.params.id);
```
Comment (stlcubsfan) [inline routes/agencies.js]: item doesn't exist...did you mean agency?

**Review by stlcubsfan — CHANGES_REQUESTED** [review]
Fix items vs agencies and I think we're good

**Comment by marcDeSantis** [comment]
OK - I made those changes

**Review by stlcubsfan — APPROVED** [review]

**Pull request merged**